docs/source/ops/connect/index.rst

```diff
@@ -0,0 +1,13 @@
+.. Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _ops-connect_index:
+
+Operating Daml Connect
+======================
+
+
+.. toctree::
+   :maxdepth: 3
+
+   content
```

Comment:
maybe rename that file to `log-and-metrics` or something? As we add further sections, having one called `content` is going to be confusing.

Reply:
I agree, will do